src/api/changePassword.js

```diff
@@ -0,0 +1,48 @@
+const router = require("express").Router();
+const nodemailer = require("nodemailer");
+const cors = require("cors");
+const User = require("../models/User");
+const bodyParser = require("body-parser");
+const jwt = require("jsonwebtoken");
+const exjwt = require("express-jwt");
+const sendResponse = require("../utils/sendResponse");
+const { SECRET_TOKEN } = require("../utils/secret-token");
+
+router.post("/changePassword", async function(req, res) {
+  if (!req.body || !req.body.email || !req.body.token || !req.body.password) {
+    sendResponse(
+      res,
+      400,
+      "Request doesn't contain all of: email, token, or password"
+    );
+    return;
+  }
+
+  const user = await User.findOne({ email: req.body.email }).catch(e =>
```

Comment:
User.findOne({ email: req.body.email, password: req.body.password }) to make sure the password also matches

Reply:
If we remove the email from the request then I'm assuming we'll just be doing the lookup by password which is the JWT.